circle.yml

```diff
@@ -7,6 +7,7 @@ dependencies:
     - sudo apt-get install libnotify4
     - sudo apt-get update
     - sudo apt-get install gvfs-bin
+    - sudo apt-get install python-nltk python-numpy
```

Comment:
self-note:
will remove this in next push (why'd I add numpy lol?)
It was failing because of `python3`.
Needs to be added to cached_dependencies.